src/org/usfirst/frc4904/robot/commands/ShootCycle.java

```diff
@@ -0,0 +1,15 @@
+package org.usfirst.frc4904.robot.commands;
+
+import org.usfirst.frc4904.robot.RobotMap;
+
+import edu.wpi.first.wpilibj.command.CommandGroup;
+
+public class ShootCycle extends CommandGroup {
+	public ShootCycle() {
+		super("ShootCycle");
+		requires(RobotMap.Component.shooter);
+		addParallel(new Align());
+		addSequential(new FlywheelSpinup());
+		addSequential(new Shoot());
```

Comment:
This is not a problem, but only because there is another problem. FlywheelSpinup extends motor constant, and thus never finishes (it is designed to be interrupted or cancelled). Therefore, the shoot will not happen prematurely because the shoot will not happen.
The FlywheelSpinup should be addParallel, because the flywheel should keep spinning up while the shoot command is running. The shoot command waits for the flywheel to be ready, so you do not need to worry about that. The Align command should be addSequential, as we want to wait for the Align to finish before we begin spinning up and shooting. Also, with power consumption it will be very difficult to spin up and turn the robot at the same time.